src/en/page-templates/page-templates.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Page templates
+layout: 'layouts/page-templates-overview.njk'
+eleventyNavigation:
+  key: pageTemplatesEN
+  title: Page Templates
```

Comment:
Just notice this, shouldn't this be "Page templates" with a lowercase 't'?

Reply:
Yes -- good catch!